### ISSUE TITLE
Option to disable on desktop

### ISSUE THREAD

**Issue by DoubleStyx** [Frozenreflex/Thundagun#46]
Right now, the mod doesn't really do anything on desktop, as there's no way to update the current perspective without moving the avatar. Since this is handled by FrooxEngine itself, it means that desktop responsiveness is tied to the engine tick rate, which this mod does not currently address. Also, the mod currently does not have a way to force the mouse input to be consumed, causing it to display over the in-game cursor (#24). #6 would make this mod viable on desktop.

Given that we don't have a good way to effectively utilize desktop mode here, it would be nice to have an option to force Unity to wait on Resonite, or maybe do this automatically based on the current mode.

**Comment by DoubleStyx** [comment]
This won't be needed with #52.
